src/ripple/app/ledger/impl/MetaView.cpp

```diff
@@ -482,23 +533,212 @@ Json::Value MetaView::getJson (int) const
 
     ret[jss::nodes] = nodes;
 
-    ret[jss::metaData] = mSet.getJson (0);
+    // VFALCO The meta only exists during apply() now
+    //ret[jss::metaData] = meta.getJson (0);
```

Comment:
No I don't know if we want to remove that because I have changed the client API here.